### ISSUE TITLE
Duplicate MODS Export Notifications

### ISSUE THREAD

**Issue by sarahjeansweeney** [NEU-Libraries/cerberus#1048]
This morning I requested a MODS export for this collection: https://repository.library.northeastern.edu/collections/neu:rx914500d

The export worked as expected, except that I received two notifications with the download link, about a minute apart.

**Comment by dgcliff** [comment]
Odd - I'll look into it

**Comment by dgcliff** [comment]
I can't seem to replicate this in development or production. Can you list the steps/collections used when this occurred?

**Comment by sarahjeansweeney** [comment]
I can't replicate this either, though I know it happened. Looking through my notification history it looks like it only happened twice:

September 26, one delivered at 9:37am and the other at 9:38am, both with the same download URL: https://repository.library.northeastern.edu/collections/neu:rx914500d
September 7, both delivered at 2:56pm with the same download URL: https://repository.library.northeastern.edu/collections/neu:cj82mt48t

I can't say for sure what steps I took to trigger the double notifications, other than that I clicked "Manage Collection", selected the Export tab, and clicked Export. I tend not to click buttons twice, though I suppose that's always possible.
